arm-logic/2016-06-01/examples/SessionsDelete.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Delete integration account session examples",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123",
+    "sessionName": "testsession123-ICN"
+  },
+  "responses": {
+    "200": {
```

Comment:
"204" is defined in the swagger but not in the example https://travis-ci.org/Azure/azure-rest-api-specs/jobs/204011630#L2425